src/s3/commands/downloadFileAs.ts

```diff
@@ -119,9 +120,17 @@ export async function downloadFileAsCommand(
     const sourcePath = readablePath(node)
 
     await telemetry.s3_downloadObject.run(async () => {
-        const saveLocation = await promptForSaveLocation(file.name, window)
+        let lastUsedDownloadPath: string | undefined
+        const storedPath = globals.context.globalState.get('workbench.explorer.downloadPath')
```

Comment:
As far as I can tell, we don't really have conventions for the key in globals.context.globalState, but would it make sense to go with something like `S3_LATEST_DOWNLOAD_PATH` or something like that?

Reply:
+1 for conventions
-1 for UPPER_CASE, idk why we are doing that anywhere for anything

Reply:
It's just codewhisperer doing the uppercase, the codebase is pretty inconsistent on it

Reply:
Going forward we should not use uppercase. That's an old habit from the original codebase that doesn't really have a purpose.